js/runtime.js

```diff
@@ -1072,7 +1098,8 @@
                 return vec.y;
             },
             randomUnitVector: function randomUnitVector(){
-                return util.Vector.fromPolar(util.randInt(0,359), 1);
+                var vec = util.Vector.fromPolar(Math.random() * 360, 1);
```

Comment:
Made the direction of the random vector more random. Now it can point in any direction, not just aligned at one-degree increments.